module/documents/combat.mjs

```diff
@@ -152,11 +152,12 @@ export default class CrucibleCombat extends foundry.documents.Combat {
         _id: impetusId,
         name: "Impetus",
         icon: "icons/magic/movement/trail-streak-zigzag-yellow.webp",
-        changes: [{key: "system.status.impetus", value: 1, mode: CONST.ACTIVE_EFFECT_MODES.OVERRIDE}],
+        statuses: ["hastened"],
         duration: {
           combat: this.id,
           rounds: 0,
-          turns: 1
+          turns: 1,
+          startRound: this.round
```

Comment:
Impetus should have a duration specified in `rounds` which expire at the beginning of the Actor's turn on the subsequent round.